### PR TITLE
Add HTS SARSCoV2 3CLPro Diamond from TDC

### PR DESCRIPTION
**Pull request by phalem**

### CODE REVIEW

**Review by phalem — COMMENTED** [review]
Add benchmark field

File: data/sarscov2_3clpro_diamond/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    A large XChem crystallographic fragment screen against SARS-CoV-2
+    main protease at high resolution. From MIT AiCures.
+targets:
+    - id: SARSCoV2_3CLPro_Diamond
+      description: activity against the SARSCoV2 3CL protease (1) or not (0)
+      units:
+      type: boolean
+      names:
+          - activity against SARSCoV2 3CL protease
+          - activity against SARS-CoV-2 3CL protease
```
Comment (kjappelbaum) [inline data/sarscov2_3clpro_diamond/meta.yaml:13]: this isn't updated yet

Comment (MicPie) [reply data/sarscov2_3clpro_diamond/meta.yaml]: just pushed it in the same minute

**Review by kjappelbaum — APPROVED** [review]
lgtm

**Pull request merged**